### PR TITLE
fix: tidy up journeys home page

### PR DESCRIPTION
**Pull request by tataihono**
# Description

### <samp>🤖 Generated by Copilot at 39dc9ce</samp>

This file updates the JourneysPage component to use modern web technologies and a new design. It enhances the user interface with `MUI` components, `Next.js` routing and localization, and journeys as embedded `iframes`. It also adds a logo and footer links to the page.

This is a qol fix for the home page. This is really clean up work.

Screenshot 2023-06-29 at 10 13 10 AM

# Walkthrough

### <samp>🤖 Generated by Copilot at 39dc9ce</samp>

*  Display journeys as iframes in a grid layout with logo and footer links ([link](https://github.com/JesusFilm/core/pull/1662/files?diff=unified&w=0#diff-7704898472a9a49a40503ef4bbac23b8f64cca8f7cb6006e59717d2f2f524be1L2-R17), [link](https://github.com/JesusFilm/core/pull/1662/files?diff=unified&w=0#diff-7704898472a9a49a40503ef4bbac23b8f64cca8f7cb6006e59717d2f2f524be1R25), [link](https://github.com/JesusFilm/core/pull/1662/files?diff=unified&w=0#diff-7704898472a9a49a40503ef4bbac23b8f64cca8f7cb6006e59717d2f2f524be1L23-R119))
  * Import components and hooks from MUI and Next.js libraries in `index.tsx` ([link](https://github.com/JesusFilm/core/pull/1662/files?diff=unified&w=0#diff-7704898472a9a49a40503ef4bbac23b8f64cca8f7cb6006e59717d2f2f524be1L2-R17))
  * Import logo image from public folder in `index.tsx` ([link](https://github.com/JesusFilm/core/pull/1662/files?diff=unified&w=0#diff-7704898472a9a49a40503ef4bbac23b8f64cca8f7cb6006e59717d2f2f524be1R25))
  * Restructure `JourneysPage` component to use `Grid`, `Image`, `Stack`, `Fade`, `Link`, and `Typography` components ([link](https://github.com/JesusFilm/core/pull/1662/files?diff=unified&w=0#diff-7704898472a9a49a40503ef4bbac23b8f64cca8f7cb6006e59717d2f2f524be1L23-R119))
  * Add animations, styles, and translations to `JourneysPage` component using `styled`, `useTranslation`, and `t` function ([link](https://github.com/JesusFilm/core/pull/1662/files?diff=unified&w=0#diff-7704898472a9a49a40503ef4bbac23b8f64cca8f7cb6006e59717d2f2f524be1L23-R119))

### CODE REVIEW

File: apps/journeys/pages/index.tsx [hunk 1]
```diff
@@ -1,40 +1,122 @@
...
+                <Link
+                  variant="body2"
+                  underline="none"
+                  target="_blank"
+                  rel="noopener"
+                >
+                  {t('Terms & Conditions')}
+                </Link>
+              </NextLink>
+              <NextLink
```
Comment (csiyang) [inline apps/journeys/pages/index.tsx:102]: There's some padding left being added to this element
Screenshot 2023-06-29 at 10 22 30 AM
Screenshot 2023-06-29 at 10 22 20 AM

File: apps/journeys/pages/index.tsx [hunk 2]
```diff
@@ -1,40 +1,122 @@
...
+                  variant="body2"
+                  underline="none"
+                  target="_blank"
+                  rel="noopener"
+                >
+                  {t('Privacy Policy')}
+                </Link>
+              </NextLink>
+              <Typography variant="body2">
+                {t('NextSteps © {{year}}', { year: new Date().getFullYear() })}
```
Comment (csiyang) [inline apps/journeys/pages/index.tsx:116]: Are we ok with this text being a different color than the 2 links?

File: apps/journeys/pages/index.tsx [hunk 3]
```diff
@@ -1,40 +1,122 @@
...
+                  <Grid
+                    item
+                    key={id}
+                    xs={12}
+                    sm={6}
+                    md={4}
+                    lg={3}
+                    sx={{ position: 'relative' }}
+                  >
+                    <Fade in timeout={(index + 1) * 1000}>
```
Comment (csiyang) [inline apps/journeys/pages/index.tsx:55]: Nice!

File: apps/journeys/pages/index.tsx (hunk 2 above)
Comment (tataihono) [reply apps/journeys/pages/index.tsx:116]: yup. otherwise this text would look like a link.

File: apps/journeys/pages/index.tsx (hunk 1 above)
Comment (tataihono) [reply apps/journeys/pages/index.tsx:102]: resolved.

**Review by csiyang — APPROVED** [review]

**Review by csiyang — APPROVED** [review]

**Pull request merged**